montage/juror_endpoints.py

```diff
@@ -152,6 +154,24 @@ def get_votes_from_round(user_dao, round_id, request, rnd=None):
     return {'data': data}
 
 
+def get_votes_stats_from_round(user_dao, round_id):
+    juror_dao = JurorDAO(user_dao)
+    rnd = juror_dao.get_round(round_id)
+
+    result = None
+
+    if rnd.vote_method == 'yesno':
+        raw_ratings = juror_dao.get_rating_stats_from_round(round_id)
+        stats = {'yes': 0, 'no': 0}
+        vote_map = {0.0: 'no', 1.0: 'yes'}
+        for count, rating_value in raw_ratings:
+            if rating_value in vote_map:
+                stats[vote_map[rating_value]] = count
+
+        result = {'method': 'yesno', 'stats': stats}
```

Comment:
would you like to extend this to support star-rating rounds, too? Also, what should the behavior be in a ranking round? Maybe just "ranked" and "unranked" counts? (unranked being the default lowest value an entry can get in a ranked round)

Reply:
This feature is mostly needed in those cases where we set up limits for the number of 'yes' votes, 5-star votes, etc. It does not apply to the ranking round, where all photos are just ranked anyway.

Reply:
1. Extended support for star-rating rounds.
2. Avoid show stats checkbox when editing ranking rounds.